### PR TITLE
Add `clearTerminal` to clear the whole terminal, not just visible part of it

### PR DESCRIPTION
**Pull request by vadimdemedes**
Extracted from https://github.com/facebook/jest/blob/1c56fb88f8527e25abffaa9de62c6e2476cdccc9/packages/jest-cli/src/constants.js#L12.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -78,6 +78,9 @@ x.scrollUp = ESC + 'S';
 x.scrollDown = ESC + 'T';
 
 x.clearScreen = '\u001Bc';
+
+// Extracted from Jest - https://github.com/facebook/jest/blob/1c56fb88f8527e25abffaa9de62c6e2476cdccc9/packages/jest-cli/src/constants.js#L12
+x.clearTerminal = process.platform === 'win32' ? `${x.eraseScreen}${ESC}0f` : `${x.eraseScreen}${ESC}3J${ESC}H`;
```
Comment (Qix-) [inline index.js]: How does the `[3J[H` help here?

**Comment by Qix-** [comment]
What's the difference? I can't imagine this clearing the scroll back buffer.

File: index.js (hunk 1 above)
Comment (vadimdemedes) [reply index.js]: No idea, just copied it from Jest's source code.

Comment (vadimdemedes) [reply index.js]: See https://github.com/facebook/jest/pull/5251.

**Comment by vadimdemedes** [comment]
> What's the difference? I can't imagine this clearing the scroll back buffer.

Yep, it's actually clearing everything, the whole session.

File: index.js (hunk 1 above)
Comment (sindresorhus) [reply index.js]: You can find something similar here: https://github.com/facebook/create-react-app/blob/master/packages/react-dev-utils/clearConsole.js

**Comment by vadimdemedes** [comment]
Hey @sindresorhus @Qix-, are there things that prevent this from merging in?

**Pull request merged**